src/reportWebVitals.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
 const reportWebVitals = onPerfEntry => {
   if (onPerfEntry && onPerfEntry instanceof Function) {
     import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
```

Comment:
- Kindly delete the webvitals file as you won't be needing it for this project 👍🏽